addons/website/views/snippets/s_banner_categories.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_banner_categories" name="Banner Categories">
+    <section
+        data-name="Banner Categories"
+        class="s_banner_categories o_cc o_cc3 oe_img_bg o_bg_img_center pb56"
+        data-oe-shape-data="{'shape':'web_editor/Connections/20','colors':{'c5':'rgb(255, 255, 255)'},'showOnMobile':false}"
+        style="background-image: url('/web/image/website.s_banner_categories_default_image_1'); background-position: 50% 75%;"
+        >
+        <div class="o_we_bg_filter bg-black-50"/>
+        <div
+            class="o_we_shape o_web_editor_Connections_20"
+            style="background-image: url('/web_editor/shape/web_editor/Connections/20.svg?c5=rgb%28255%2C255%2C255%29');"
+            />
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11" style="gap: 15px;">
+                <div
+                    data-name="Intro"
+                    class="o_grid_item o_colored_level g-height-6 g-col-lg-8 order-0"
+                    style="z-index: 1; --grid-item-padding-y: 50px; grid-area: 2 / 3 / 8 / 11;"
+                    >
+                    <h1 class="pt32" style="text-align: center;">Explore Our New Collection</h1>
+                    <p class="lead" style="text-align: center;">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus venenatis rhoncus justo quis fringilla.
+                    </p>
+                    <p style="text-align: center;">
+                        <a
+                            t-att-href="cta_btn_href"
+                            class="btn btn-lg btn-secondary o_translate_inline bg-white border-0 mt32"
```

Comment:
See [this comment](https://github.com/odoo/odoo/pull/209241#discussion_r2084171689)

Tip: usually we use `br` tags to add space between inline elements (see `s_split_intro`)